bundle/manifests/rc.app.stacks_runtimecomponents.yaml

```diff
@@ -1033,6 +1033,8 @@ spec:
                         type: string
                     type: object
                 type: object
+              disableServiceLinks:
```

Comment:
A description should be added:
Disable information about services being injected into the application pod's environment variables. Default to false.